src/pages/dashboard.tsx

```diff
@@ -16,22 +16,21 @@ export const getServerSideProps = (async (context) => {
   );
   if (loginProps?.isAuthorized) {
     const supabase = getSupabase(loginProps.userID);
-    const { data } = await supabase
+    const { data, error } = await supabase
       .from("users")
       .select()
       .eq("passage_user_id", loginProps.userID);
-    console.log(data);
     return {
       props: {
         isAuthorized: loginProps.isAuthorized,
-        data: data,
+        userID: loginProps.userID
```

Comment:
On main we have the props for AuthProps as `isAuthorized` and `data` which we get back from supabase, and then we are using that in the other components to get the users info. Is that still how it will work? I see that data was removed here, but I think I am a little confused on how we get the users info for the other pages if we only return userID in `serversideprops` and not data?

Reply:
In my most recent push, I combined them so the pages that need data get data props and the pages that need userID get userID; I was just used to using userID from the dashboard when verifying the user from Passage that I stuck with that, but it works both ways!